lib/ClangImporter/ClangClassTemplateNamePrinter.cpp

```diff
@@ -129,6 +129,12 @@ struct TemplateInstantiationNamePrinter
 
     return buffer.str().str();
   }
+
+  std::string VisitVectorType(const clang::VectorType *type) {
+    return (Twine("SIMD") + std::to_string(type->getNumElements()) + "<" +
```

Comment:
I think with twine we might no longer need the `to_string`. But not 100% confident on that.

Reply:
It doesn't compile for me if I remove `to_string`.

Reply:
Yeah, I think I slightly prefer `to_string` over `Twine(...)` in this case, since that makes it explicit that we aren't e.g. constructing a string with a given capacity/size – something a `std::string` constructor would do 🙂